src/domains/transaction/components/SecondSignatureRegistrationForm/VerificationStep.tsx

```diff
@@ -12,24 +15,56 @@ export const VerificationStep = () => {
 	const [defaultMnemonic] = useState(() => watch("secondMnemonic"));
 	const mnemonic = getValues("secondMnemonic") || defaultMnemonic;
 
+	const [mnemonicValidated, setMnemonicValidated] = useState(false);
+	const [passphraseDisclaimer, setPassphraseDisclaimer] = useState(false);
+
 	const { t } = useTranslation();
 
 	const handleComplete = (isComplete: boolean) => {
-		setValue("verification", isComplete, { shouldDirty: true, shouldValidate: true });
+		setMnemonicValidated(isComplete);
 	};
 
+	useEffect(() => {
+		setValue("verification", passphraseDisclaimer && mnemonicValidated, {
+			shouldDirty: true,
+			shouldValidate: true,
+		});
+	}, [mnemonicValidated, passphraseDisclaimer]);
+
 	useEffect(() => {
 		register("verification", { required: true });
 	}, [register]);
 
+	useEffect(() => {
```

Comment:
This is here to make testing easier, I'll remove this once testing is done

fyi @samharperpittam